monocle/main.py

```diff
@@ -131,7 +134,8 @@ def main():
                     loop_delay=project['crawler']['loop_delay'],
                     token=crawler_item['token'],
                     base_url=crawler_item['base_url'])
-                tpool.append(Crawler(c_args, elastic_conn=args.elastic_conn))
+                tpool.append(Crawler(c_args, elastic_conn=args.elastic_conn,
+                                     elastic_timeout=args.elastic_timeout))
```

Comment:
You need that change on the gerrit crawler too, few lines below.

Reply:
done